addons/mail/static/src/core/web/chatter.js

```diff
@@ -379,7 +415,7 @@ export class Chatter extends Component {
         this.closeSearch();
         const schedule = async (threadId) => {
             await this.activityService.schedule(this.props.threadModel, [threadId]);
-            this.load(this.props.threadId, ["activities", "messages"]);
+            this.load(this.props.threadModel, this.props.threadId, ["activities", "messages"]);
```

Comment:
I think state is the currently displayed there, so we should be consistent and never use props?